scripts/storage-emulator-integration/tests.ts

```diff
@@ -1067,27 +1067,18 @@ describe("Storage emulator", () => {
 
           const requestClient = TEST_CONFIG.useProductionServers ? https : http;
           await new Promise((resolve, reject) => {
-            requestClient.get(
-              downloadUrl,
-              {
-                headers: {
-                  // This is considered an authorized request in the emulator
-                  Authorization: "Bearer owner",
-                },
-              },
-              (response) => {
-                const data: any = [];
-                response
-                  .on("data", (chunk) => data.push(chunk))
-                  .on("end", () => {
-                    expect(Buffer.concat(data)).to.deep.equal(
-                      Buffer.from(IMAGE_FILE_BASE64, "base64")
-                    );
-                  })
-                  .on("close", resolve)
-                  .on("error", reject);
-              }
-            );
+            requestClient.get(downloadUrl, (response) => {
```

Comment:
Do you not need to pass an empty object as the options?

Reply:
no, `get` has two versions, a 3 arg version that takes options and 2 arg version that takes just the path and callback